Description: Disable unavailable optional feature and dep, nt-time
 not in Debian, for Windows (NTFS) anway.
Forwarde: not-needed
Author: Blair Noctis <ncts@debian.org>
Last-Update: 2025-03-24
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -102 +102 @@
-nt-time = ["dep:nt-time"]
+#nt-time = ["dep:nt-time"]
@@ -181 +181 @@
-[dependencies.nt-time]
+[disabled.dependencies.nt-time]
